### PR TITLE
Update with new glider sequence, addition of new oceanographic element and new code table entries for GTSPP quality flag.

### PR DESCRIPTION
**Pull request by david-i-berry**
Update with new glider sequence, addition of new oceanographic element and new code table entries for GTSPP quality flag.

issue #16

### CODE REVIEW

**Review by amilan17 — DISMISSED** [review]

File: BUFR_TableD_en_15.csv [hunk 1]
```diff
@@ -293,6 +293,64 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,004025,Time period or displacement,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014017,Instantaneous long-wave radiation,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014018,Instantaneous short-wave radiation,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301150,WIGOS identifier,,,79,Operational
```
Comment (amilan17) [inline BUFR_TableD_en_15.csv:296]: Keep or remove note id 79?

File: BUFR_TableD_en_15.csv [hunk 2]
```diff
@@ -293,6 +293,64 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
...
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,022032,Speed of sea-surface current,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,022005,Direction of sea-surface current,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301011,"Year, month, day",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301013,"Hour, minute, second",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,008021,Time significance,"Set to 2, time averaged",,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,004025,Time period or displacement,"Duration of dive, e.g. -10 (minutes)",,,Operational
```
Comment (amilan17) [inline BUFR_TableD_en_15.csv:319]: "e.g. -10 (minutes)"

Can a time duration be negative?

File: BUFR_TableD_en_15.csv [hunk 3]
```diff
@@ -293,6 +293,64 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,004025,Time period or displacement,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014017,Instantaneous long-wave radiation,,,,Operational
 15,Oceanographic report sequences,315011,(Met-ocean observations from autonomous surface vehicles),,014018,Instantaneous short-wave radiation,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301150,WIGOS identifier,,,79,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,201129,Change data width,Add one bit to width of following descriptor,,,Operational
```
Comment (amilan17) [inline BUFR_TableD_en_15.csv:297]: sequence name differs from issue sumary: 

"Ocean glider trajectory profile sequence" vs  "Sequence for reporting observations from a single glider trajectory profile"

File: BUFR_TableD_en_15.csv (hunk 1 above)
Comment (david-i-berry) [reply BUFR_TableD_en_15.csv]: Cut and paste error, sorry. I've now corrected it in the latest commit.

File: BUFR_TableD_en_15.csv (hunk 2 above)
Comment (david-i-berry) [reply BUFR_TableD_en_15.csv:319]: Yes, this is a time displacement.

File: BUFRCREX_TableB_en_42.csv [hunk 4]
```diff
@@ -14,3 +14,4 @@ ClassNo,ClassName_en,FXY,ElementName_en,BUFR_Unit,BUFR_Scale,BUFR_ReferenceValue
 42,Oceanographic elements,042013,a2 coefficient of the directional Fourier series,Numeric,4,-20000,15,Numeric,4,6,,,Operational
 42,Oceanographic elements,042014,b2 coefficient of the directional Fourier series,Numeric,4,-20000,15,Numeric,4,6,,,Operational
 42,Oceanographic elements,042015,Check factor K,Numeric,2,0,12,Numeric,2,4,,,Operational
+42,Oceanographic elements,042016,Sea water potential density referenced to sea surface,Numeric,5,-10000000,24,Numeric,5,9,,,Operational
```
[the remark this answers is not in the document]
Comment (david-i-berry) [reply BUFRCREX_TableB_en_42.csv]: Now corrected.

File: BUFR_TableD_en_15.csv (hunk 3 above)
Comment (david-i-berry) [reply BUFR_TableD_en_15.csv:297]: I changed the name to be consistent with naming of the other sequences but we can use that originally proposed.

Comment (amilan17) [reply BUFR_TableD_en_15.csv:297]: Ok. I'll update the summary to reflect the branch

**Review by amilan17 — APPROVED** [review]

**Pull request merged**